nix/pkgs/control-plane/cargo-project.nix

```diff
@@ -47,27 +47,28 @@ let
   LIBCLANG_PATH = "${llvmPackages.libclang}/lib";
   PROTOC = "${protobuf}/bin/protoc";
   PROTOC_INCLUDE = "${protobuf}/include";
-
+  src_folders = [
```

Comment:
Maybe we shouldn't use "folders". `Cargo.lock` & `Cargo.toml` aren't folders.

Reply:
Good point! `src_list` ?